packages/web/src/shared/Button.3.tsx

```diff
@@ -165,26 +163,28 @@ interface Props {
 function ButtonPrimary(props: Props) {
   const { children, status, size, style, href, target, onDarkBackground } = props
   return (
-    <Text
-      href={href}
-      target={target}
-      accessibilityRole="link"
-      style={[
-        baseStyles.base,
-        baseStyles.verticallyAlign,
-        fonts.navigation,
-        sizeStyle(size),
-        primaryStyles[status],
-        textStyles.medium,
-        status === BTNStates.disabled && onDarkBackground
-          ? primaryStyles.darkText
-          : primaryStyles.text,
-        style,
-        status === BTNStates.disabled && baseStyles.notAllowed,
-      ]}
-    >
-      {children}
-    </Text>
+    <Link href={href} passHref={true}>
+      <Text
```

Comment:
when        `accessibilityRole="link"` `Text` is render as anchor, `Link` sets some function handlers on that anchor so that for local hrefs a full page load is not initated